style.css

```diff
@@ -611,3 +620,807 @@ body {
   list-style: none;
   padding-left: 80px;
 }
+
+/* Desktop version */
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] I noticed that your design has some issues that you will need to fix. Please make your website adapt well to any kind and size of desktop screen. You can see the issues with the video recording that I'm sharing with you below :arrow_down: It's really important to pay attention to the details.

https://www.loom.com/share/0b8f8edf20794ad68fb931f1ca3a24c0

Reply:
I have implemented the given feedback.

Reply:
- [ ] Kindly make sure you fix the responsiveness issue as suggested by the previous reviewers.
As you can you how your design is looking :point_down: 
![Screenshot from 2022-09-30 16-50-37](https://user-images.githubusercontent.com/97100091/193298288-5723230f-c0ed-4f8f-b516-982011bb0696.png)
- [ ] Kindly make sure you fix this issue
[here](https://www.loom.com/share/e81ca027adb84fcdab6113f47d54e730) you can see how your page is responding on differents screen size